src/dbnode/integration/commitlog_bootstrap_only_reads_required_files_test.go

```diff
@@ -33,9 +33,11 @@ import (
 )
 
 func TestCommitLogBootstrapOnlyReadsRequiredFiles(t *testing.T) {
-	if testing.Short() {
-		t.SkipNow() // Just skip if we're doing a short run
-	}
+	// TODO(rartoul): Temporarily disabled until a subsequent P.R that will
+	// improve and simplify the commitlog bootstrapping logic. This is fine
+	// because this integration test protects against performance regressions
+	// not correctness.
+	t.SkipNow()
```

Comment:
Ok cool, can we open an issue for this and link to it from here?  Just so we're tracking it as an issue too?

Reply:
created a master task with all commitlog rotation / snapshotting remaining work: #1383